timetracker/vms/templates/vms/dashboard.html

```diff
@@ -4,13 +4,14 @@
   <h1>Dashboard</h1>
   <h2>Quick Links</h2>
   {% if request.user.is_staff %}
-    <a href="{% url 'vms:create-staff-agency' %}">create a staffing agency!</a>
+    <a href="{% url 'vms:create-staff-agency' %}">create a staffing agency!</a><br>
   {% endif %}
+  <a href="{% url 'vms:create-staff-employee' %}">apply as a staffing agency employee!</a><br>
   {% if staff_admin %}
-    <h2>Staff List</h2>
+    <h2>Staff</h2>
     <ul>
         {% for staff in staff_admin %}
-          <li>{{ staff.agency }}</li>
+          <li><a href= "{{ staff.agency.get_absolute_url }}">{{ staff.agency }}</a></li>
```

Comment:
The changes to the listing of staffing agencies the user is an admin for have already been made in the master branch, so look out for that when rebasing.